projects/metis/src/app/_helpers/cache.ts

```diff
@@ -13,23 +13,26 @@ export class SingleCache<Value> {
     if (this.observable && !refresh) {
       return this.observable;
     }
-    const observable = (this.observable = this.sourceFn().pipe(
+    const observable = this.sourceFn().pipe(
```

Comment:
Why not directly assign it to this.observable? Doesn't look like we need the local variable.

Reply:
We agreed that this is not possible, and added comments (and better unit tests) to that effect.